models/fates/inst/create_1x1_ref_case.sh

```diff
@@ -52,7 +57,7 @@ echo "*** Modifying xmls  ***"
 ./xmlchange -file env_mach_pes.xml -id MAX_TASKS_PER_NODE -val 1
 ./xmlchange -file env_mach_pes.xml -id TOTALPES -val 1
 # Modifying : env_build.xml
-./xmlchange -file env_build.xml -id CIME_OUTPUT_ROOT -val ${CASEROOT}
+./xmlchange -file env_build.xml -id CESMSCRATCHROOT -val ${CASEROOT}
```

Comment:
Im still uncertain about this because it is my understanding that CESMSCRATCHROOT is an old variable and as of a few re-factors ago it is now CIME_OUTPUT_ROOT.  This is what I use, not scratch root and it works for me.  So again this is unclear. I pointed this out to Ryan and he didnt really know...he was just having you try various things.  This should be clarified.

Reply:
For example, @tonygardella look at the config_machines.  In there the XML tag is CIME_OUTPUT_ROOT and used to be CESMSCRATCHROOT that is why I think that var has changed, but again Ryan wasn't clear about that/

Reply:
Yeah, take a look in here: https://github.com/NGEET/fates-clm/blob/master/cime/config/cesm/machines/config_machines.xml

Reply:
That link doesn't work for me. Is it a private repo? 

That being said, a grep for "CESMSCRATCHROOT" only can find it in "ngeet-config_machines.xml". I'll switch it back to "CIME_OUTPUT_ROOT".

Reply:
Yeah its private, I thought you had access.  But yeah try with CIME_OUTPUT_ROOT in your create run script and see if you can then case.submit the model and that it works.....

Reply:
Tried it with just the CIME_OUTPUT_ROOT and it worked fine so I  removed CESMSCRATCHROOT.